### ISSUE TITLE
Windows Terminal not responding

### ISSUE THREAD

**Issue by waltonereed** [jupyter/notebook#3454]
I'm running jupyterlab 0.31.4 on a Windows 10 64 bit machine. When I try launch a terminal session, I get the following screen:
![image](https://user-images.githubusercontent.com/25827972/37720748-ebc4223a-2cf5-11e8-9f05-5651d0a4e6c7.png)

I've tried various keyboard entries to get the terminal active but nothing works. Has anyone encountered this issue before?

**Comment by takluyver** [comment]
cc @blink1073 @ccordoba12

**Comment by ccordoba12** [comment]
@waltonereed, is it working for you in the classic notebook?

**Comment by waltonereed** [comment]
@ccordoba12 I'm not sure what you mean...I can run terminal in Jupyter Notebook?

**Comment by jasongrout** [comment]
Yes - in the /tree listing, click "New" in the upper right and select Terminal.

**Comment by waltonereed** [comment]
My apologies, I never run terminal from Jupyter. No it does work in Notebook or Labs.

**Comment by jasongrout** [comment]
> No it does work

did you mean to say "does *not* work"?

This helps narrow down the issue (unfortunately puts it out of my experience too...)

**Comment by ccordoba12** [comment]
@andfoy, this seems like a problem in pywinpty. What code can @waltonereed run in a console to verify if that's the case?
